sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1343,4 +1343,13 @@ export class ApplicationService extends RecordDataModelService<Application> {
       )
       .getOne();
   }
+  async getApplicationByApplicationNumberStudent(
+    applicationNumber: string,
+    student: Student,
+  ): Promise<Application> {
+    return this.repo.findOne({
+      applicationNumber: applicationNumber,
+      student: student,
```

Comment:
Can a student can submit or upload files for any application status?

Reply:
good qn. yes since here `ApplkicationNumber` is only metadata, @abschwenker said not to consider the status of the application